services/oauth/client.go

```diff
@@ -0,0 +1,26 @@
+package oauth
+
+import (
+	. "google-scraper/helpers"
+
+	"github.com/go-oauth2/oauth2/v4/models"
+	"github.com/google/uuid"
+)
+
+func GenerateOauthClient(userID int64) (client *models.Client, err error) {
+	clientId := uuid.New().String()
+	clientSecret := uuid.New().String()
+
+	client = &models.Client{
+		ID:     clientId,
+		Secret: clientSecret,
+		UserID: IntToString(userID),
```

Comment:
@junan What's your purpose of using this `UserID`? 🤔
Is this use for filtering for just only client belonging to that user?

Reply:
Junan will keep this for his future task 🙌